### DISCUSSION TITLE
I have not received my education benefits within the past 2 days

### DISCUSSION THREAD

**Discussion by alexCobat1212** [community/community#145455] [GitHub Education] [answered]
### Hi everyone,

I hope you can assist me, as I have not received my education benefits within the past 2 days. I would greatly appreciate any help or guidance on this matter.
Can you please Help
Thanks

**Comment by davevad93** [comment, marked as answer]
Hi @alexCobat1212 ,

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @alexCobat1212 was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

A `PRO` badge will also appear on your profile.

<img width="250" alt="00" src="https://github.com/user-attachments/assets/f4f6eed9-f10c-45b9-b792-b055ec1ecd2c">

**Reply by alexCobat1212** [reply]
Appreciate Bro
